include/picongpu/param/laser.param

```diff
@@ -1,728 +0,0 @@
-/* Copyright 2013-2022 Axel Huebl, Anton Helm, Rene Widera, Richard Pausch,
- *                     Alexander Debus, Nils Prinz, Klaus Steiniger
- *
- * This file is part of PIConGPU.
- *
```

Comment:
@PrometheusPi The removed laser.param is already in the PR, I thought I forgot it but it is here.
All other laser.param were removed before in the rPRs where we switched to incident field lasers.